code/datums/station_traits/neutral_traits.dm

```diff
@@ -127,3 +127,49 @@
 
 	var/new_colored_assistant_type = pick(subtypesof(/datum/colored_assistant) - get_configured_colored_assistant_type())
 	GLOB.colored_assistant = new new_colored_assistant_type
+
+/datum/station_trait/cargorilla
+	name = "Cargo Gorilla"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 1
+	show_in_report = FALSE // Selective attention test. Did you spot the gorilla?
+	/// The gorilla we created
+	var/mob/living/simple_animal/hostile/gorilla/cargo_domestic/cargorilla
+
+/datum/station_trait/cargorilla/New()
+	. = ..()
+	RegisterSignal(SSatoms, COMSIG_SUBSYSTEM_POST_INITIALIZE, .proc/replace_cargo)
+
+/datum/station_trait/cargorilla/proc/replace_cargo(datum/source)
+	SIGNAL_HANDLER
+
+	var/mob/living/simple_animal/sloth/cargo_sloth = locate(/mob/living/simple_animal/sloth/citrus) || locate(/mob/living/simple_animal/sloth/paperwork)
```

Comment:
When the

Reply:
it's only ran once on atom init what's the worst that could happen + punpun trait does it to + L

Reply:
watch this RATIO

Reply:
>locate in world
no pls

Reply:
where were the complainers when headcoder ninjanomnom added a locate world with the punpun station trait

i'll change it to globals